[circle-opselector] Create CLI Skeleton (#7777)
Samsung/ONE: compiler/circle-opselector/README.md

# circle-opselector

> `circle-opselector` is a tool for creating new circle model by various methods to select nodes.

Reviewer: ```suggestion
`circle-opselector` is a tool for creating new circle models by selecting nodes from a model.

```
